api/src/main/java/org/openmrs/module/referencemetadata/OrderFrequencyMetaData.java

```diff
@@ -0,0 +1,120 @@
+        public static final String EVERY_FIVE_HOURS = "162248OFAAAAAAAAAAAAAAA";
+        public static final String EVERY_SIX_HOURS = "162249OFAAAAAAAAAAAAAAA";
+        public static final String EVERY_EIGHT_HOURS = "162250OFAAAAAAAAAAAAAAA";
+        public static final String EVERY_TWELVE_HOURS = "162251OFAAAAAAAAAAAAAAA";
+        public static final String EVERY_24_HOURS = "162252OFAAAAAAAAAAAAAAA";
+        public static final String EVERY_48_HOURS = "162253OFAAAAAAAAAAAAAAA";
+        public static final String EVERY_36_HOURS = "162254OFAAAAAAAAAAAAAAA";
+        public static final String EVERY_72_HOURS = "162255OFAAAAAAAAAAAAAAA";
+        public static final String MONDAY_WEDNESDAY_FRIDAY = "162256OFAAAAAAAAAAAAAAA";
+    }
+
+    public static class _Concept {
+        public static final String ONCE = "162135";
+        public static final String TWICE_DAILY = "160858";
+        public static final String TWICE_DAILY_BEFORE_MEALS = "160859";
+        public static final String TWICE_DAILY_AFTER_MEALS = "160860";
+        public static final String TWICE_DAILY_WITH_MEALS = "160861";
+        public static final String ONCE_DAILY = "160862";
+        public static final String ONCE_DAILY_BEDTIME = "160863";
+        public static final String ONCE_DAILY_EVENING = "160864";
+        public static final String ONCE_DAILY_MORNING = "160865";
+        public static final String THRICE_DAILY = "160866";
+        public static final String THRICE_DAILY_AFTER_MEALS = "160867";
+        public static final String THRICE_DAILY_BEFORE_MEALS = "160868";
+        public static final String THRICE_DAILY_WITH_MEALS = "160869";
+        public static final String FOUR_TIMES__DAILY_WITH_MEALS = "160870";
+        public static final String FOUR_TIMES__AFTER_WITH_MEALS_BEDTIME = "160871";
+        public static final String FOUR_TIMES__BEFORE_WITH_MEALS_BEDTIME = "160872";
+        public static final String EVERY_30_MINS = "162243";
+        public static final String EVERY_HOUR = "162244";
+        public static final String EVERY_TWO_HOURS = "162245";
+        public static final String EVERY_THREE_HOURS = "162246";
+        public static final String EVERY_FOUR_HOURS = "162247";
+        public static final String EVERY_FIVE_HOURS = "162248";
+        public static final String EVERY_SIX_HOURS = "162249";
+        public static final String EVERY_EIGHT_HOURS = "162250";
+        public static final String EVERY_TWELVE_HOURS = "162251";
+        public static final String EVERY_24_HOURS = "162252";
+        public static final String EVERY_48_HOURS = "162253";
+        public static final String EVERY_36_HOURS = "162254";
+        public static final String EVERY_72_HOURS = "162255";
+        public static final String MONDAY_WEDNESDAY_FRIDAY = "162256";
+    }
+
+    @Override
+    public void install() throws Exception {
+        install(orderFrequency(_Concept.ONCE, _OrderFrequency.ONCE, 0.001));
```

Comment:
Flagging this so we can discuss whether 0.001 is correct for this, or how else to model it.

Reply:
Hmmm, it was a tricky situation to express. I thought about 0.0 but then I realised it has to take a valid frequency. Please I need suggestions on how best to model this

Reply:
Yes, please ask this on talk.